paddle/fluid/platform/mkldnn_reuse.h

```diff
@@ -542,6 +562,73 @@ class ConvMKLDNNTemplateHandler : public MKLDNNHandler {
                                scale_data, mask);
   }
 
+  mkldnn::primitive_attr CreatePostOps(bool fuse_relu,
```

Comment:
@jczaja I have no idea about why these code moving can improve the performance, It seems that the creation of primitive descriptors was not decreased as I commented on "conv_mkldnn_op.cc", could you please explain it?

Reply:
@xiaolil1 Just moving this code does not improve performacne. PD reusing improves performance. CreatePostOps was present in conv as well as in conv_transpose, and now instead of two instances we have only one in mkldnn_reuse. So that particular change is refactoring to avoid redundant code. See some comments above